config/externalnamenottested.go

```diff
@@ -219,7 +219,7 @@ var ExternalNameNotTestedConfigs = map[string]config.ExternalName{
 	// which is an attribute reported in the state.
 	// TODO: we need to check the value of a global_replication_group_id to
 	// see if further normalization is possible
-	"aws_elasticache_global_replication_group": config.IdentifierFromProvider,
+	// "aws_elasticache_global_replication_group": config.IdentifierFromProvider,
```

Comment:
As I can see in the uptest run, the `global_replication_group_id` is the suffix we provided `example` and contains the `virxk` value generated by the cloud provider: `virxk-example`. In this case, normalization is not possible, please remove all lines related to the GlobalReplicationGroup resource (including line 218) .

Reply:
Remove commented code